libdns.go

```diff
@@ -219,10 +349,15 @@ func RelativeName(fqdn, zone string) string {
 // prepending it to zone and tidying up the dots. For example, an input
 // of name "sub" and zone "example.com." will return "sub.example.com.".
 func AbsoluteName(name, zone string) string {
+	if name == "" {
+		// This should trigger an error, but the function signature doesn't
+		// allow for that. I guess return an empty string?
+		return ""
+	}
 	if zone == "" {
 		return strings.Trim(name, ".")
 	}
-	if name == "" || name == "@" {
+	if name == "@" {
 		return zone
 	}
```

Comment:
Should I change this function signature to return an error if `""` is passed, or just return the empty string `""` as I am right now, or something else?

Reply:
Isn't it okay if the name is empty string? It just means the FQDN is the zone, right?

Reply:
That goes back to [`"@"` vs `""` discussion](https://github.com/libdns/libdns/pull/152#discussion_r1989557976), and it would seem weird to me to allow `""` here but forbid it in the `Record` struct. Or maybe these are different cases since the struct field gets initialized to `""` but you always need to explicitly pass in a function parameter?